tests/test_ingesters.py

```diff
@@ -174,6 +176,8 @@ class MetanormIngesterTestCase(django.test.TestCase):
     """Test the base metadata ingester class"""
 
     def setUp(self):
+        Parameter.objects.count = mock.Mock()
+        Parameter.objects.count.return_value = 2
```

Comment:
Please use patch for this, in order to limit unintended side effects

Reply:
Done in the next commit

Reply:
Done in the next commit